tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/IastSettingsTests.cs

```diff
@@ -70,6 +70,28 @@ public void GivenIastSettings_WhenSetMaxConcurrentRequestsToMinus1_MaxConcurrent
         Assert.Equal(IastSettings.MaxConcurrentRequestDefault, iastSettings.MaxConcurrentRequests);
     }
 
+    [Fact]
+    public void GivenIastSettings_WhenMaxRangeCountTo5_MaxRangeCountIsDefaultValue()
+    {
+        var settings = new CustomSettingsForTests(new Dictionary<string, object>()
+        {
+            { ConfigurationKeys.Iast.MaxRangeCount, 5 }
+        });
+        var iastSettings = new IastSettings(settings, NullConfigurationTelemetry.Instance);
+        Assert.Equal(5, iastSettings.MaxRangeCount);
+    }
+
+    [Fact]
+    public void GivenIastSettings_WhenSetMaxRangeCountToMinus1_MaxRangeCountIsDefaultValue()
+    {
+        var settings = new CustomSettingsForTests(new Dictionary<string, object>()
+        {
+            { ConfigurationKeys.Iast.MaxRangeCount, -1 }
```

Comment:
I would also add a test for `0`, since the validation code doesn't allow zero.
Or maybe you could allow zero if it makes sense. Now sure what this is used for, so I can't say 😝

Reply:
I added a test for 0 then :)